src/control/content/components/Popup.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
```

Comment:
why do we have this heere?

Reply:
I was trying to make the popup component work, so I tried not to use the script and make it this way instead. Used another component in the end, this is not needed. I will remove it and redeploy the PR